x-pack/plugins/enterprise_search/public/applications/app_search/utils/results/index.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const flattenField = (fieldName: string, fieldValue: object): Array<[string, object]> => {
```

Comment:
Suggestion: use `Record<string, unknown>` instead of `object`. Ideally we'd be using generics and some type mapping functions here to retain the typing as much as possible, but that frankly seems like a lot of effort for not much gain.

Reply:
I have no idea how I would rewrite this with generics. Could you give an example?